packages/webpack-cli/lib/webpack-cli.js

```diff
@@ -1363,10 +1363,21 @@ class WebpackCLI {
                     );
                 }
 
+                const flag = this.getBuiltInOptions().find(
+                    (flag) => option.long === `--${flag.name}`,
+                );
+
+                if (flag && flag.configs[0].values) {
```

Comment:
I think we can have more them, i.e. `config[1]`/`config[2]`, I think better use `map` here and collect them

Reply:
Will update 👍